packages/cryptoplease/pubspec.yaml

```diff
@@ -60,6 +60,7 @@ dependencies:
   solana:
     path: ../solana/
   sqlite3_flutter_libs: ^0.5.10
+  timelines: ^0.1.0
```

Comment:
We try to minimize the number of dependencies, especially when it comes to widgets. This should be pretty easy to implement on our own, so let's rather create a component in `cryptoplease_ui`.

It's even better to make it a separate PR.

Reply:
Understood. Have created a separate PR for this widget. 

Will update this PR once other one is merged.

Reply:
Removed `timelines` dependency and replaced it with custom `CpTimeline`.